master/reference/felix/configuration.md

```diff
@@ -37,6 +37,8 @@ The full list of parameters which can be set is as follows.
 | ReportingIntervalSecs (FELIX_REPORTINGINTERVALSECS)                      | Interval at which Felix reports its status into the datastore or 0 to disable.  Must be non-zero in OpenStack deployments. [Default: `30`]                                                       | int                                     |
 | ReportingTTLSecs (FELIX_REPORTINGTTLSECS)                                | Time-to-live setting for process-wide status reports. [Default: `90`]                                                                                                                            | int                                     |
 | IpInIpMtu (FELIX_IPINIPMTU)                                              | The MTU to set on the tunnel device. See [Configuring MTU]({{site.baseurl}}/{{page.version}}/usage/configuration/mtu) [Default: `1440`]                                                          | int                                     |
+| IpInIpEnabled (FELIX_IPINIPENABLED)                                      | When Kubernetes datastore driver is being used, tell Felix if IPinIP is enabled (not required for Etcd datastore since this information is available through the datastore) [Default: `false`]   | boolean                                 |   
```

Comment:
@kprabhak I think the description is a little bit off for the reference docs - it should just say "Whether or not Felix should configure an IPIP tunnel interface on the host"

The other bits of info - kdd vs etcd - is just backstory for why it's set a certain way in our manifests. (It can be set through the datastore via KDD now, for example)